core/backend/src/GeoCoordServices.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { IModelJsNative } from "@bentley/imodeljs-native";
+import { Range2dProps } from "@itwin/core-geometry";
+
+/** Class containing services for working with geographic coordinates.
+ * @public
```

Comment:
id rather we start off with `@beta` and work up to public

Reply:
Now using @beta